node_cli/operations/common.py

```diff
@@ -82,11 +80,8 @@ def configure_flask():
         logger.info('Flask secret key generated and saved')
 
 
-def configure_iptables():
-    try:
-        run_cmd(['bash', CONFIGURE_IPTABLES_SCRIPT])
-    except Exception:
-        error_msg = 'iptables configure script failed'
-        logger.exception(error_msg)
-        error_exit(error_msg, exit_code=CLIExitCodes.SCRIPT_EXECUTION_ERROR)
-    logger.info('iptables configured successfully')
+def unpack_backup_archive(backup_path: str) -> None:
```

Comment:
It's possible to use context manager here. Let's do it.

Reply:
done